.github/workflows/python-package.yml

```diff
@@ -3,7 +3,7 @@
 
 name: Python package
 
-on: [pushes]
+on: [push]
```

Comment:
Very good.
A question: How do we specify this should only runs when a pull request is merged?

Reply:
According to [events-that-trigger-workflows](https://docs.github.com/en/actions/reference/events-that-trigger-workflows) I can add/change the event to pull_request (closed)